python/cuml/test/test_linear_model.py

```diff
@@ -488,3 +488,82 @@ def test_logistic_predict_convert_dtype(train_dtype, test_dtype):
     clf = cuLog()
     clf.fit(X_train, y_train)
     clf.predict(X_test.astype(test_dtype))
+
+
+@pytest.fixture(scope='session',
+                params=['binary', 'multiclass'])
+def regression_dataset(request):
+    regression_type = request.param
+    n_samples = 500000
+    n_features = 5
```

Comment:
perhaps it would be good to generate stress tests and perhaps a case with larger n_features